Makefile

```diff
@@ -404,3 +404,11 @@ integration-tests: all testrunner
 #     integration-tests
 #
 	RUNNER_ARGS="-parallel" ./test/run-integration-tests.sh
+
+netlify-site-preview:
+	@mkdir -p site-build
+	@curl --user $(WEBSITE_GITHUB_USER) --silent 'https://codeload.github.com/weaveworks/website-next/tar.gz/$(WEBSITE_BRANCH)' \
```

Comment:
Both, `WEBSITE_GITHUB_USER` and `WEBSITE_BRANCH` are in Netlify configuration. I'm using tarball API, as far as I know it's faster then `git clone` (even when you pass `--depth 10 --single-branch`), but happy to change if we have a good reason.